native/src/routes/CityNotCooperating.tsx

```diff
@@ -80,8 +79,10 @@ const TemplateText = styled.Text`
 
 const CityNotCooperating = (): ReactElement => {
   const { t } = useTranslation('cityNotCooperating')
+  const { cityNotCooperatingIcon } = buildConfigAssets()
   const [isCopied, setIsCopied] = useState<boolean>(false)
   const theme = useTheme()
+  const template: string = buildConfig().featureFlags.cityNotCooperatingTemplate!
```

Comment:
```suggestion
  const template = buildConfig().featureFlags.cityNotCooperatingTemplate
```

Reply:
For this kind of assertion the route should only be enabled if the template is not null. I.e. use the following in the navigator:
```
        const cityNotCooperatingEnabled = template && icon
```
and then
```
        {cityNotCooperatingEnabled && <Stack.Screen name={CITY_NOT_COOPERATING_ROUTE} component={CityNotCooperating} />}
```
Then it is fine imo to use non null assertion all the time.